packages/react-reconciler/src/ReactFiberThrow.new.js

```diff
@@ -244,6 +247,7 @@ function throwException(
     }
 
     if (enableSchedulingProfiler) {
+      markComponentRenderStopped();
```

Comment:
so suspending is considered as render stopped, at least for the purposes of what we show in dev tools?

Reply:
Yup!